webpack 5 install 'url' and 'events' as a dependency

<!--
  Please note that this template is not optional.
  Please fill out _ALL_ fields, or your issue may be closed as "invalid."
  Please do not delete this template.
  Please ask questions on StackOverflow or Gitter (https://gitter.im/webpack/webpack).
  General questions, how-to questions, and support requests will be closed.
-->

* Operating System: Mac OS
* Node Version: v10.11.0
* NPM Version: 6.4.1
* webpack Version: 5
* webpack-dev-server Version: 3.1.10

<!--
  Please place an x, no spaces, in all [ ] that apply
  Please note that we are NOT accepting FEATURE requests at this time.
-->

- [x] This is a **bug**
- [ ] This is a **modification** request

### Code

<!--
 for webpack 5 - install the "url"  and "events" as a dependency as using alias doesn't get rid of the errors
-->

```js
  // webpack.config.js
var path = require('path');

module.exports = {
    //...
    devServer: {
        contentBase: path.join(__dirname, 'dist'),
        compress: true,
        port: 9000,
    },
    resolve: {
        alias: {
            'url': false,
            'events' : false
        }
    }
};
```

### Expected Behavior

### Actual Behavior
```js
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need these module and configure a polyfill for it.

If you want to include a polyfill, you need to install 'url'.
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.alias: { "url": false }

ERROR in (webpack)/hot/emitter.js 1:19-36
Module not found: Error: Can't resolve 'events' in '/Users/debbieobrien/webpack/node_modules/webpack/hot'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need these module and configure a polyfill for it.

If you want to include a polyfill, you need to install 'events'.
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.alias: { "events": false }
 @ (webpack)-dev-server/client?http://localhost:9000 226:21-51
ℹ ｢wdm｣: Failed to compile.
```
after adding the alias to the webpack.config I get this error
```js
✖ ｢wds｣: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.resolve.alias should be one of these:
   object { <key>: string } | [object { alias?, name?, onlyModule? }]
   -> Redirect module requests
   Details:
    * configuration.resolve.alias['url'] should be a string.
      -> New request
    * configuration.resolve.alias['events'] should be a string.
      -> New request
    * configuration.resolve.alias should be an array:
      [object { alias?, name?, onlyModule? }]

```

### For Bugs; How can we reproduce the behavior?
Run the webpack-dev-server in next branch

### For Features; What is the motivation and/or use-case for the feature?

--- a/package.json
+++ b/package.json
@@ -51,6 +51,7 @@
     "spdy": "^4.0.0",
     "strip-ansi": "^3.0.0",
     "supports-color": "^5.1.0",
+    "url": "^0.11.0",
     "webpack-dev-middleware": "3.4.0",
     "webpack-log": "^2.0.0",
     "yargs": "12.0.2"

